opportunity/views.py

```
@@ -89,7 +89,7 @@ def get_context_data(self, **kwargs):
         context["opportunities"] = opportunities
         context["accounts_list"] = AccountSerializer(accounts, many=True).data
         context["contacts_list"] = ContactSerializer(contacts, many=True).data
-        context["tags"] = TagsSerailizer(Tags.objects.filter(), many=True).data
+        context["tags"] = TagsSerailizer(Tag.objects.filter(), many=True).data
```

Ruff (0.8.2)

`Tag` may be undefined, or defined from star imports

(F405)